src/core/store/columnContext.tsx

```diff
@@ -0,0 +1,121 @@
+import { useContext, createContext } from 'react';
+import type { MutableRefObject } from 'react';
+
+export const ColumnContext = createContext<Resizer | string>('no provider was given');
```

Comment:
I also can't find a reason to have string there. We don't put there other strings, only one place here. 
Maybe just null? And there `if (typeof columnContext === 'string') return;` process as null

Reply:
i was hoping it would give better readability, but since you've asked, it didn't - so i'll change it